backend/app/requests/adequacy.py

```diff
@@ -61,18 +61,20 @@ def adequacy_request(app, flask_request, engine):
     except json.JSONDecodeError:
         raise InvalidFormat(message='Invalid JSON format.')
 
-    if 'provider_ids' not in request:
-        raise InvalidFormat(message='Invalid format. Could not find provider_ids.')
+    if 'providers' not in request:
+        raise InvalidFormat(message='Invalid format. Could not find provider information.')
     if 'service_area_ids' not in request:
         raise InvalidFormat(message='Invalid format. Could not find service_area_ids.')
 
-    provider_ids = request['provider_ids']
+    if 'providers' in request:
```

Comment:
If providers not in request, you raise an error above. Is there a need to check again?

Reply:
Done